New logic to allow altering of SpySat status via [Upgrade] (#1062)
Phobos-developers/Phobos: CREDITS.md

  - Laser trails for VoxelAnims
  - MakeInfantry logic on BombClass bugfix
  - Debris & meteor impact behaviour settings

>   - Fixed broken Building Upgrade logic for SpySat=yes

Reviewer: ```suggestion
  - Upgrade logic to allow altering of SpySat status
```
